code/src/hooks/use-update-cart-item-quantity/useUpdateCartItemQuantity.ts

```diff
@@ -10,9 +16,15 @@ const useUpdateCartItemQuantity = () => {
   const updateQuantity = async (params: CartManagementPatchParams) => {
     try {
       await updateCartItemQuantity(params).unwrap();
-    } catch {
+    } catch (err) {
+      const apiError = err as ApiErrorQuantity;
+
+      //@TODO Change logic handle products items from the stock for quantity

```

Comment:
What does this TODO mean?

Reply:
> What does this TODO mean?

I'll remove it when productQuantity will be ready from the backend side
and rewrite some logic